tests/bdd/features/drain/node/feature.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Drain
+
+  Background:
+    Given Multiple uncordoned nodes
+
+  Scenario: Draining a node
+    Given an uncordoned node
+    When the user issues a drain command to the node
+    Then the node should be cordoned
+    And nexuses should be moved to other suitable nodes
+
+  Scenario: Draining a cordoned node
+    Given a cordoned node
+    When the user issues a drain command to the node
+    Then the node should remain cordoned
+    And nexuses should be moved to other suitable nodes
+
+# The system is unable to move all resources from the draining node to another node
+  Scenario: Draining a node with insufficient system capacity
+    Given an uncordoned node
+    And insufficient system-wide capacity on other nodes to accommodate all drained resources
+    When the user issues a drain command to the node
+    Then the drain should fail to complete in its entirety
```

Comment:
Do we need a use case for the system resources changing during the drain? Eg: initially we have enough capacity, but as we're draining, that changes for some reason (example: more volumes were created, or a node is lost, or even cordoned)

Reply:
Yes good idea. I have added an additional scenario. Can you take a look and let me know what you think?